src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,4 +1,10 @@
 package frc.robot;
 
+import com.flash3388.flashlib.frc.robot.RoboRio;
+import com.flash3388.flashlib.hid.HidChannel;
+
 public class RobotMap {
+    public static final HidChannel XBOX = RoboRio.newHidChannel(0);
+    public static final int SHOOTER_MASTER = 0; // Need to set the number to the correct one.
```

Comment:
generally, if you give yourself comments about what needs to be done, start them with `// TODO:`.
It will be highlighted by _Intellij_ and be easier to spot.